.circleci/config.yml

```diff
@@ -166,6 +166,21 @@ jobs:
       #    is triggered automatically via GitHub webhook on https://readthedocs.org/projects/git-machete/builds/,
       # 3) every time a https://github.com/r-ryantm bot runs on NixOS/nixpkgs repository,
       #    Nix git-machete package is updated with the fresh version/package hash according to the data from PyPI.
+
+  "Publish Dockerfile to Docker Hub":
+    working_directory: /git-machete
+    docker:
+      - image: docker:17.05.0-ce-git
+    steps:
+      - checkout
+      - setup_remote_docker
+      - deploy:
+          name: Publish application to docker hub
+          command: |
+            docker login -e $DOCKER_HUB_EMAIL -u $DOCKER_HUB_USER_ID -p $DOCKER_HUB_PWD
```

Comment:
Pls take a look at https://github.com/VirtusLab/git-machete/blob/master/ci/docker-build-and-push.sh#L19 , the credentials are already provided in the CI... also, password should rather be passed on stdin rather than via command line

Reply:
Please check again if it is good now.